.github/workflows/dart-ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     # available. See https://hub.docker.com/r/google/dart/ for the available
     # images.
     container:
-      image: google/dart:latest
+      image: google/dart:beta
```

Comment:
Is this still necessary?

Dart SDK stable has reached `2.12.0` version with the release of Flutter `2.0` I think, which implies that `beta` might not be necessary anymore.